src/m365/spo/commands.ts

```diff
@@ -25,6 +25,7 @@ export default {
   CONTENTTYPE_FIELD_REMOVE: `${prefix} contenttype field remove`,
   CONTENTTYPE_FIELD_SET: `${prefix} contenttype field set`,
   CONTENTTYPE_REMOVE: `${prefix} contenttype remove`,
+  CONTENTTYPE_LIST: `${prefix} contenttype list`,
```

Comment:
Let's put the reference in alphabetical order to make it easier to look them up in the future

Reply:
done